bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -202,6 +202,10 @@ public void stopPackage(String packageName)
     public String getCurrentPackage() throws AndroidDebugBridgeDeviceException, InterruptedException,
             AndroidDebugBridgeDeviceReadException, TimeoutException, ExecutionException {
         var out = runAdbShell("dumpsys", "window", "windows", "|", "grep", "mFocusedApp");
+        if (getAndroidVersion().contains("11")) {
```

Comment:
I think we should not rely on this call, this way we are adding an extra adb command per execution for an information that is already stored in the thing properties. 
We should instead pass the android version stored on the device properties to this class.

Reply:
oh ? where is it stored? I was kind of confused on that cuz I see in the mDNS discovery it seems to define that but I swear it wasn't a prop before

Reply:
so how do I access that from the AndroidDebugBridgeDevice class?

Reply:
It's described in the previous comment:
| I think you could just add a 'setVersion(String version)' method to the 'AndroidDebugBridgeDevice' class and call it from this one, so the version is propagated to the 'AndroidDebugBridgeDevice' instance each time the handler calls 'refreshProperties' and stored in a private property that we can rely on.

So my proposal is adding a @Nullable private String variable called version in the 'AndroidDebugBridgeDevice', adding there a public void setVersion(String version) to setting it from the 'refreshProperties' method in the handler.

Reply:
OK. I don't know how to do that (or I might but not know the terminology). Would you be up for doing that ? or show me an example of how to? Or if you want you can just push the change...